### ISSUE TITLE
Feedback BB | 09.06.19

### ISSUE THREAD

**Issue by beachbagenal** [FAC-Sixteen/RE4.W#15]
Hi guys, so here is some feedback.

Generally, everyone I have spoken to really likes it.

I think the colour scheme, level of language is right.

Pages:

Register, login, play game (p.1)

- [ ] sign up button should be first

View story (p.11 / 17)

- [ ] I think it would be good to have a page after the first page which explains what a landfill is and how long things take to decompose (cans - 100 years, plastic - 1,000 years, glass - 1,000,000 years ). In the UK it is estimated that we only have 7 years left of capacity of landfills. So it's important we recycle things that can be recycled!

Adventure mode / game (p.19)

- [ ] I really like having 5 different products and the user having to drag the item into the right bin with a time limit. I think that there should be an instant answer and points when an item is put in the bin. Then after the 5 items have been done, the second phase should be a multiple choice questions about the different products, e.g. a question about how many times around the world the soft drink cans could go, and then another one on plastic etc.

Choose your hero (p.2)

- [ ] This is really cool, are we OK to use these icons or are they licensed?

Let me know if you have any questions, otherwise speak on Tuesday.

**Comment by sylv1ah** [comment]
Hi Beach!

Thanks for logging the feedback! 

The first two points can be easily adjusted. 

For the adventure mode/game part would it be good to instead of having a multiple choice question it would be a true or false question given that many kids may not know the answer of these questions to decide between a list of choices where as a true or false may be easier to comprehend? 
E.g Soft drink cans alone can wrap around the circumference of the world, true or false? False! It can wrap around 3 times! 

About the icons, you are able to use them but there would need to be a place where we need to give credit. We got them from this site: https://www.flaticon.com/

We will be starting the build today if there's any other questions or issues that may come up we will be communicating it on this issue #8 , likewise if there's any other questions/issues on your end to be put there too!

**Comment by beachbagenal** [comment]
Yep agree, let's build do a multiple choice version first. When do you think it would be available to test?

On the icons, that is totally cool. Would you suggest a glossary page or something like that for the credit?

**Comment by Fweddi** [comment]
Given the time constraints, we will probably only be able to develop one game for the MVP. I think it might be better to prioritise the drag-and-drop game idea (i.e. whether the items can be recycled or not) over the multiple choice questions, as it is probably more educational for kids to learn the basic facts. We can still include the interesting facts as we planned (appearing on the next page), rather than in game form.

**Comment by beachbagenal** [comment]
Yes that should be the priority for sure.
